src/uwtools/drivers/mpas_init.py

```diff
@@ -94,8 +94,9 @@ def namelist_file(self):
         path = self._rundir / fn
         yield asset(path, path.is_file)
         yield None
-        duration = timedelta(hours=self._driver_config["boundary_conditions"]["length"])
-        str_duration = str(duration).replace(" days, ", "")
+        stop_time = self._cycle + timedelta(
+            hours=self._driver_config["boundary_conditions"]["length"]
+        )
         try:
```

Comment:
This also looks to be similar to what is in the `mpas` driver. Should we update the other as well?

Reply:
No, that was the problem. These two tools have distinctly different namelists. We went one step two far in making them similar.

Reply:
That makes sense!